.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,160 @@
+name: benchmarks
+
+on:
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron: '0 */8 * * *' # Run workflow threee times a day
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+
+jobs:
+  benchmarks_windows:
+    strategy:
+      matrix:
+        test_script: [NuGetClient-win-d76a117]
+      fail-fast: false
+    runs-on: windows-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: code
+
+      - name: Benchmark
+        shell: pwsh
+        run: |
+          .\code\scripts\perftests\testCases\${{matrix.test_script}}.ps1
+          Rename-Item -Path "results.csv" -NewName "${{matrix.test_script}}.csv"
+          
+      - name: Upload benchmark results
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{matrix.test_script}}
+          path: ${{matrix.test_script}}.csv
+          
+  benchmarks_linux:
+    strategy:
+      matrix:
+        test_script: [LargeAppCPM64-142722b, LargeAppCPM64-nostaticgraph-142722b, OrchardCore-5dbd92c, Orleans-eda972a]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: code
+
+      - name: Benchmark
+        shell: pwsh
+        run: |
+          .\code\scripts\perftests\testCases\${{matrix.test_script}}.ps1
+          Rename-Item -Path "results.csv" -NewName "${{matrix.test_script}}.csv"
+          
+      - name: Upload benchmark results
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{matrix.test_script}}
+          path: ${{matrix.test_script}}.csv
+
+  process_data:
+    # Only run on main repository
+    # Scheduled workflows do not have information about fork status, hence the hardcoded check
+    if: github.repository == 'G-Research/NuPerfMonitor' && github.ref == 'refs/heads/main'
```

Comment:
We have to change the repository name here.... right ? @marcin-krystianc

Reply:
Yes, you are right. I'll leave it as it is for now and will let you update all the places referring to the old repo.